jkube-kit/resource/helm/src/main/java/org/eclipse/jkube/kit/resource/helm/HelmService.java

```diff
@@ -229,8 +235,65 @@ private static void splitAndSaveTemplate(Template template, File templatesDir) t
     }
   }
 
-  static void createChartYaml(HelmConfig helmConfig, File outputDir) throws IOException {
-    final Chart chart = new Chart();
+  static void createChartYaml(HelmConfig helmConfig, File outputDir, JKubeConfiguration jKubeConfiguration) throws IOException {
+    final Chart chartFromHelmConfig = createChartFromHelmConfig(helmConfig);
```

Comment:
minor: if the methods implementations are in the same file, have the implementations in the same order of appearance.